src/PrestaShopBundle/Form/Exception/InvalidConfigurationDataError.php

```diff
@@ -52,17 +52,17 @@ class InvalidConfigurationDataError
      * @param string $fieldName
      * @param int|null $languageId
      */
-    public function __construct(int $errorCode, string $fieldName, ?int $languageId = null)
```

Comment:
I have changes errorCode to string i.e. `error_not_numeric_or_lower_than_zero` to make error codes more unique. With this PR I realized that having error codes as 1,2, 3 could be very problematic when you try to return error by error code but you have multiple dataProviders which then must have different error codes.